src/Rest/Helpers/Command/CommandBuilder.php

```diff
@@ -194,4 +195,59 @@ public function get(): array
 
         return $data;
     }
+
+    public function matchesApplicationCommand(ApplicationCommand $applicationCommand): bool
+    {
+        $applicationCommandBuilder = self::fromApplicationCommand($applicationCommand)->get();
+
+        foreach ($this->get() as $key => $value) {
+            if ($applicationCommandBuilder[$key] !== $value) {
+                return false;
+            }
+        }
+
+        return true;
+    }
+
+    protected static function fromApplicationCommand(ApplicationCommand $applicationCommand): self
+    {
+        $commandBuilder = new self();
+
+        $commandBuilder->setDefaultMemberPermissions(
+            Bitwise::from((int)$applicationCommand->default_member_permissions)
```

Comment:
```suggestion
            Bitwise::from((int) $applicationCommand->default_member_permissions)
```
Isn't `default_member_permisssions` a bit list or whatever discord calls it? If so, there's a separate static method to initialize from that.